Lib/_pydatetime.py

```diff
@@ -1827,7 +1827,7 @@ def now(cls, tz=None):
     def utcnow(cls):
         "Construct a UTC datetime from time.time()."
         import warnings
-        warnings.warn("datetime.utcnow() is deprecated and scheduled for "
+        warnings.warn("datetime.datetime.utcnow() is deprecated and scheduled for "
                       "removal in a future version. Instead, Use timezone-aware "
```

Comment:
Fix capitalisation / consistency with C messages
```suggestion
                      "removal in a future version. Use timezone-aware "
```